packages/parrot-devtools/src/app/index.js

```diff
@@ -12,6 +12,7 @@
  * the License.
  */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
 import 'whatwg-fetch';
```

Comment:
Any way we don't have extraneous dependencies exception?

Reply:
updated in 824f96d I added to deps.